standard/ERCs/erc-6900.md

```diff
@@ -171,35 +209,13 @@ interface IStandardExecutor {
     /// @param calls The array of calls.
     /// @return An array containing the return data from the calls.
     function executeBatch(Call[] calldata calls) external payable returns (bytes[] memory);
-}
-```
-
-#### `IModuleExecutor.sol`
-
-Execution interface for calls made from modules. Modular Smart Contract Accounts **MUST** implement this interface to support execution from modules.
-
-The `executeFromModuleExternal` function SHOULD check whether the call's target implements the `IModule` interface via ERC-165.
 
-**If the target of `executeFromModuleExternal` function is a module, the call SHOULD revert.**
-
-This prevents accidental misconfiguration or misuse of modules (both installed and uninstalled). Installed modules MAY interact with other installed modules via the `executeFromModule` function.
-
-```solidity
-interface IModuleExecutor {
-    /// @notice Execute a call from a module through the account.
-    /// @dev Permissions must be granted to the calling module for the call to go through.
+    /// @notice Execute a call using a specified runtime validation, as given in the first 21 bytes of
+    /// `authorization`.
     /// @param data The calldata to send to the account.
-    /// @return The return data from the call.
-    function executeFromModule(bytes calldata data) external payable returns (bytes memory);
-
-    /// @notice Execute a call from a module to a non-module address.
-    /// @dev If the target is a module, the call SHOULD revert. Permissions must be granted to the calling module
-    /// for the call to go through.
-    /// @param target The address to be called.
-    /// @param value The value to send with the call.
-    /// @param data The calldata to send to the target.
-    /// @return The return data from the call.
-    function executeFromModuleExternal(address target, uint256 value, bytes calldata data)
+    /// @param authorization The authorization data to use for the call. The first 21 bytes specifies which runtime
```

Comment:
Same here - also probably want to update the source comment to use 24 instead of 21